### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by yusufyusie**
- Create a table named owners with the following columns:

      id: integer (set it as autoincremented PRIMARY KEY)
      full_name: string
      age: integer

- Create a table named species with the following columns:

        id: integer (set it as autoincremented PRIMARY KEY)
        name: string

- Modify animals table:

      Make sure that id is set as autoincremented PRIMARY KEY
      Remove column species
      Add column species_id which is a foreign key referencing species table
      Add column owner_id which is a foreign key referencing the owners table

- Insert the following data into the owners table:

          Sam Smith 34 years old.
          Jennifer Orwell 19 years old.
          Bob 45 years old.
          Melody Pond 77 years old.
          Dean Winchester 14 years old.
          Jodie Whittaker 38 years old.

- Insert the following data into the species table:

          Pokemon
          Digimon

- Modify your inserted animals so it includes the species_id value:

          If the name ends in "mon" it will be Digimon
          All other animals are Pokemon

- Modify your inserted animals to include owner information (owner_id):

            Sam Smith owns Agumon.
            Jennifer Orwell owns Gabumon and Pikachu.
            Bob owns Devimon and Plantmon.
            Melody Pond owns Charmander, Squirtle, and Blossom.
            Dean Winchester owns Angemon and Boarmon.

- Write queries (using JOIN) to answer the following questions:
- What animals belong to Melody Pond?

image

- List of all animals that are pokemon (their type is Pokemon).

image

- List all owners and their animals, remember to include those that don't own any animal.

image

- How many animals are there per species?

image

- List all Digimon owned by Jennifer Orwell.

image

- List all animals owned by Dean Winchester that haven't tried to escape.

image

- Who owns the most animals?

image

### CODE REVIEW

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @yusufyusie, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Well structured project ✅ 

## Required Changes ♻️
- [ ] 🚀 Impressive work! Your queries are well-structured and aligned with the project requirements. However, I'd like to offer some suggestions to enhance your project's documentation in the `README`:

1. 📚 **Remove Placeholder Texts:** To provide clarity to your users, consider removing all placeholder texts from your `README`. These placeholders are intended for you, the developer, to guide in writing descriptive and structured documentation.
Screenshot 2023-10-07 at 09 38 35

2. 🚦 **Get Started Section:** In the "Get Started" section, it's crucial to fill in the right information for the following subsections: `Install`, `Usage`, `Run test`, and `Deployment`. These sections are essential for users who want to use your project. 

3. 📝 **Customization:** If sections like `Deployment`, `Live demo`, `Run test`, or `FAQ` don't apply to your project, feel free to remove them to avoid any confusion and keep the documentation concise.

Happy coding! 😊👍📝🌟

   
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (Alusp) [inline README.md:1]: - [ ] You have done a really good job in implementing the changes requested by the previous reviewer, however, as e mentioned earlier, please ensure that you remove all place holders, below are te list of place olders you sould remove.

1. > Mention all of the collaborators of this project.
2. > Describe 1 - 3 features you will add to the project.
3. > Give credit to everyone who inspired your codebase.
4. `_NOTE: we recommend using the [MIT license](https://github.com/yusufyusie/Vet-clinic/blob/clinic/LICENSE) -` `you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._`

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @yusufyusie, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
close

 ## Highlights
✔️ Good PR Summary. ✔️
 ✔️ Completed most of the TASKS ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @yusufyusie , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Nice code :+1: 
Great effort in making the requested changes :+1: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**